index.js

```diff
@@ -261,10 +261,10 @@ class CoverageSonar extends CoverageBase {
      *                                          to talk to coverage server
      */
     _getAccessToken({ annotations, buildCredentials }) {
-        const { jobId, pipelineId } = buildCredentials;
+        const { jobId, pipelineId, prParentJobId } = buildCredentials;
         const { projectKey, username } = getProjectData({
             enterpriseEnabled: sonarEnterprise,
-            jobId,
+            jobId: prParentJobId || jobId,
```

Comment:
won't this change behavior for non-enterprise Sonar ? I would say pass `prParentJobId` separately and let `getProjectData` compute value.

Reply:
moved